File: android/src/main/java/com/getcapacitor/community/classes/results/IsFingerprintValidResult.java
package com.getcapacitor.community.classes.results;

import com.getcapacitor.JSObject;
import com.getcapacitor.community.interfaces.Result;

public class IsFingerprintValidResult implements Result {

    private final Boolean isValid;

    public IsFingerprintValidResult(Boolean isValid) {
        this.isValid = isValid;
    }

    @Override
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        if (isValid != null) result.put("isValid", isValid);

        return result;
    }
}

File: ios/Sources/BridgefyPlugin/Classes/Results/IsFingerprintValidResult.swift
import Foundation
import Capacitor

@objc public class IsFingerprintValidResult: NSObject, Result {
    let isValid: Bool

    init(_ isValid: Bool) {
        self.isValid = isValid
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()

        result["isValid"] = isValid

        return result as AnyObject
    }
}
